time_tracker_api/time_entries/time_entries_namespace.py

```diff
@@ -23,11 +23,18 @@
         description='The id of the selected activity',
         example=faker.random_int(1, 9999),
     ),
-    'technologies': fields.String(
-        required=True,
-        title='Technologies',
-        max_length=64,
-        description='Canonical names of the used technologies during this period',
+    'technologies': fields.List(
+        fields.String(
+            required=True,
+            title='Technologies',
+            max_length=64,
+            description='Canonical names of the used technologies during this period',
```

Comment:
is this description more clear for you?

**Technology names used in this time-entry**